builder/vsphere/common/storage_config.go

```diff
@@ -65,49 +86,32 @@ import (
 //	    "disk_controller_index": 1
 //	  }
 //	],
-//
 // ```
 //
-// In HCL2:
-// ```hcl
-//
-//	disk_controller_type = ["pvscsi", "pvscsi"]
-//	storage {
-//	   disk_size = 15000,
-//	   disk_controller_index = 0
-//	}
-//	storage {
-//	   disk_size = 15000
-//	   disk_controller_index = 0
-//	}
-//	storage {
-//	   disk_size = 15000
-//	   disk_controller_index = 1
-//	}
-//	storage {
-//	   disk_size = 15000
-//	   disk_controller_index = 1
-//	}
-//
-// ```
+
```

Comment:
Superfluous empty line? I may be wrong but I suspect this will unlink the above documentation from the type, is that expected?

Reply:
Addressed in ccab453.